pkg/state/filter.go

```diff
@@ -68,5 +69,5 @@ func filterOut(currentState shared.State, interfacesFilterGlob glob.Glob) (share
 		return currentState, err
 	}
 
-	return shared.State{Raw: filteredState}, nil
+	return shared.NewState(RemoveDynamicAttributes(string(filteredState))), nil
```

Comment:
The commit message does not match the content here, does it?

Reply:
The commit message from this line is `Filter out routes and dynamic attributes` I would say that it's correct.